### PR TITLE
Add tier filtering to dashboard

### LINKED ISSUE #252: Add filtering of students by intervention tier in addition to the type.
" Is there any way to disaggregate the phone calls, home visits, and letters by who needs "letter 1" vs "letter 2" for example?"

-- Jennifer

### PR DESCRIPTION
**Pull request by dting**
Resolves #252.

@pdotsani

### CODE REVIEW

File: client/app/app.scss [hunk 1]
```diff
@@ -58,3 +58,8 @@ body,
     width: 100%;
   }
 }
+
+// Makes the outline on focus less intrusive.
+:focus {
+  outline: thin dotted !important;
+}
```
Comment (pdotsani) [inline client/app/app.scss]: What do you think about removing this all together? I feel like our UI is very intuitive as is, maybe down the road we can remove this style on elements that don't necessarily need to be highlighted when focused.

Comment (dting) [inline client/app/app.scss]: The focus outline is for being able to use the application vis keyboard. If it is removed there is no indication that an element focused making keyboard input impossible. We could make custom style for focus highlighting at some point.

http://www.outlinenone.com/

**Comment by dting** [comment]
Oops. Sorry i committed this PR. Were there any other issues?

**Comment by pdotsani** [comment]
Nope all good. Thanks for the clarification.

**Pull request merged**